plugins/resources/unixfilesystem/src/unixfilesystem.cpp

```diff
@@ -202,101 +206,89 @@ auto unix_file_start_operation(irods::plugin_property_map& prop_map) -> irods::e
 // NOTE: All storage resources must do this on the physical path stored in the file object and then update
 //       the file object's physical path with the full path
 
-static irods::error unix_file_copy(
-    const int mode,
-    const char* srcFileName,
-    const char* destFileName)
+static irods::error unix_file_copy(const int mode, const char* srcFileName, const char* destFileName)
 {
     struct stat statbuf;
-    int status = stat( srcFileName, &statbuf );
+    int status = stat(srcFileName, &statbuf);
     int err_status = errno;
-    if ( status < 0 ) {
-        return ERROR(UNIX_FILE_STAT_ERR, fmt::format(
-            "stat failed on \"{}\", status: {}",
-            srcFileName, err_status));
+    if (status < 0) {
+        return ERROR(UNIX_FILE_STAT_ERR, fmt::format("stat failed on \"{}\", status: {}", srcFileName, err_status));
     }
 
-    if ( ( statbuf.st_mode & S_IFREG ) == 0 ) {
-        return ERROR(UNIX_FILE_STAT_ERR, fmt::format(
-            "srcFileName \"{}\" is not a regular file.",
-            srcFileName, err_status));
+    if ((statbuf.st_mode & S_IFREG) == 0) {
+        return ERROR(UNIX_FILE_STAT_ERR,
+                     fmt::format("srcFileName \"{}\" is not a regular file.", srcFileName, err_status));
     }
 
     int inFd{-1};
     int outFd{-1};
-    const irods::at_scope_exit close_fds{
-        [&]()
-        {
-            if (inFd > 0) close(inFd);
-            if (outFd > 0) close(outFd);
-        }
-    };
-
-    inFd = open( srcFileName, O_RDONLY, 0 );
+    const irods::at_scope_exit close_fds{[&]() {
+        if (inFd > 0)
+            close(inFd);
+        if (outFd > 0)
+            close(outFd);
```

Comment:
please add open/close braces around this... seems dangerous...

Reply:
Done